libyul/optimiser/StackLimitEvader.h

```diff
@@ -59,7 +59,7 @@ class StackLimitEvader
 	/// are contained in a recursive function.
 	static void run(
 		OptimiserStepContext& _context,
-		Object& _object,
+		Block& _object,
```

Comment:
It's not an object :)

```suggestion
		Block& _astRoot,
```